transaction.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN;
+UPDATE animals SET species='unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth>'2022-01-01';
+SAVEPOINT dateOfBirth;
+UPDATE animals SET weight_kg=-1*weight_kg;
+ROLLBACK TO dateOfBirth;
+UPDATE animals SET weight_kg=-1*weight_kg WHERE weight_kg<0;
+COMMIT; 
```

Comment:
**Optional Changes**
- [ ] Good effort with transactions. There is no need to create an extra file `transaction.sql` though. Please let us put this back into the `queries.sql` before merging the changes.

Reply:
Thank you for letting me know. I put transactions on queries.sql.